minitest-display.gemspec

```diff
@@ -5,7 +5,7 @@
 
 Gem::Specification.new do |s|
   s.name = "minitest-display"
-  s.version = "0.1.0"
+  s.version = "0.1.1"
```

Comment:
Instead of updating the `patch` version as new `patch` version you introduced may already be existing for the gem.

Why not increment the `build` or `revision number`?

For example:

`0.1.0.1`

Reply:
This was mentioned in the tech context to update the version number to `0.1.1`.

Reply:
Sorry, I have updated in the tech context too.